### PR TITLE
Disable command override if a bukkit/forge hybrid is detected.

### PR DESCRIPTION
**Pull request by Lyfts**
Command override is very incompatible with bukkit and a plugin should be used to handle command permission instead. 

This disables the feature for Crucible, Thermos and hopefully Ultramine. I was only able to find the source for [Ultramine 1.8](https://github.com/TechCatOther/ultramine_core) so lets hope the classpath hasn't changed.

Closes: https://github.com/GTNewHorizons/ServerUtilities/issues/12

### CODE REVIEW

File: src/main/java/serverutils/ServerUtilities.java [hunk 1]
```diff
@@ -134,14 +133,15 @@ public void onServerStarted(FMLServerStartedEvent event) {
         if (Ranks.isActive()) {
             Ranks.INSTANCE.commands.clear();
 
-            boolean crucibleLoaded = Loader.isModLoaded("Crucible");
+            boolean bukkitLoaded = CommonUtils.getClassExists("thermos.ThermosRemapper")
```
Comment (mitchej123) [inline src/main/java/serverutils/ServerUtilities.java:136]: Another common one is "org.bukkit.World"

**Review by mitchej123 — APPROVED** [review]

File: src/main/java/serverutils/ServerUtilities.java (hunk 1 above)
Comment (bombcar) [reply src/main/java/serverutils/ServerUtilities.java:136]: Would it also make sense to have a cmdfile override to shut the feature off so other examples of these could be manually handled by cfg ? cmdoverride=off kinda thing

Comment (mitchej123) [reply src/main/java/serverutils/ServerUtilities.java:136]: `-DforceBukkit=true`

Comment (Lyfts) [reply src/main/java/serverutils/ServerUtilities.java:136]: Bukkit path added aswell.

@bombcar cmd override can already be turned off in config if that's what you mean.

**Pull request merged**